setup.py

```diff
@@ -4,9 +4,36 @@
 
 VERSION = 'v0.5.0'
 
-long_desc = '''
-This package contains the Jupyter Sphinx extension.
-'''
+LONG_DESCRIPTION = """
+This package contains a `Sphinx <http://www.sphinx-doc.org/en/master/>`_ for compiling 
```

Comment:
missing the word "extension" in "contains a Sphinx extension"

Reply:
thanks @slel fixed.